pkg/scheduler/apis/config/types.go

```diff
@@ -87,6 +87,12 @@ type KubeSchedulerConfiguration struct {
 	// the default value (10s) will be used.
 	PodMaxBackoffSeconds int64
 
+	// PodMaxInUnschedulablePodsDuration is the maximum time a pod can stay in
+	// unschedulablePods. If a pod stays in unschedulablePods for longer than this
+	// value, the pod will be moved from unschedulablePods to backoffQ or activeQ.
+	// If this value is empty, the default value (24h) will be used.
```

Comment:
explain a little bit why 24h is used by default.

Reply:
Ideally, we should just get rid of the flush completely.  And we have changed it from `60s` to `5min` since #108761.
There are no more bugs were reported. So, it should be safe to set it to a larger value, such as 24h.
Of course, whether or not to use 24h can be discussed more.

Reply:
set it as 24h is `~=` never flush at all, it's hard to image a pod cannot be scheduled and stay in the  `unschedulablePods` for larger than 24h.

Reply:
I saw the suggestion from @alculquicondor from that issue.

> [Extreme] Get rid of the flush, since we don't have evidence...

Maybe he can help to judge this.